src/BenchmarkManager.py

```diff
@@ -56,6 +114,18 @@ def __init__(self, fail_fast: bool = False):
         self.results = []
         self.store_dir = None
         self.benchmark_record_template = None
+        self.interrupted_results_path = None
+
+    def load_interrupted_results(self) -> list:
+        """
+        :return: the content of the results file from the QUARK run to be resumed or None.
```

Comment:
If the output can be X or None, the return type should be "Optional[X]". In this case, Optional would need to be imported from typing alongside List and Dict in line 24.